src/client/entrypoints/main/main.tsx

```diff
@@ -0,0 +1,89 @@
+import React from 'react';
+import {useEffect, useState} from 'react';
+import ReactDOM from 'react-dom';
+import {Hideable} from '_client/common/ui_components/hideable';
+
+import {setupEditorPanel} from '_client/entrypoints/main/board_tools';
+import {
+  setupActiveBoard,
+  setupChatPanel,
+} from '_client/entrypoints/main/game_board';
+import {checkDefined} from '_common/preconditions';
+
+export const MAIN_BOARD_STUB = 'mainBoard';
+
+export enum NavbarOption {
+  MAIN = 'Game Board',
+  EDITOR = 'Board Tools',
+}
+
+ReactDOM.render(<Panels />, document.querySelector('#contentStub'));
```

Comment:
Similar question on the two `.renders` here

Reply:
so, I'm not entirely sure how to avoid this one. HTML body is basically just 

<body>
  <div id="contentStub"></div>
</body>

And Panels contains all the skeleton components that everything relies on. Does that make sense?

Reply:
However, I did add Navbar directly to Panels.

Reply:
Ah ok, yes that makes sense. And this matches what I've seen in the past. There's one "real" HTML file and then a render puts the main TSX file on it and the react components handle themselves from there.